Swift/Tests/SampleCodeTest.swift

```diff
@@ -536,5 +536,80 @@ class SampleCodeTest: CBLTestCase {
 
         print("\(config)")
     }
+    
+    func dontTestGettingStarted() throws {
+        // # tag::getting-started[]
+        // Get the database (and create it if it doesn’t exist).
+        let database: Database
+        do {
+            database = try Database(name: "mydb")
+        } catch {
+            fatalError("Error opening database")
+        }
+        
+        // Create a new document (i.e. a record) in the database.
+        let mutableDoc = MutableDocument()
+            .setFloat(2.0, forKey: "version")
+            .setString("SDK", forKey: "type")
+        
+        // Save it to the database.
+        do {
+            try database.saveDocument(mutableDoc, concurrencyControl: .lastWriteWins)
```

Comment:
For simplicity, should we call the default database.save(mutableDoc) instead? Do you intentionally want to call with concurrencyControl parameter?